include/oneapi/math/rng/detail/armpl/onemath_rng_armpl.hpp

```diff
@@ -0,0 +1,56 @@
+/*******************************************************************************
+* Copyright 2025 SiPearl
+* Copyright 2020-2021 Intel Corporation
```

Comment:
Could you please clarify why the Intel's copyright is added?

Reply:
hello, most of these files have been duplicated from oneMKL backend, as implementation is roughly the same (OpenRNG from ArmPl is a drop-in replacement for Intel VSL, using same API). So initial copyright was retained in these files. I can remove it if necessary, but I see this as derivative, so apache license says to retain them.

Reply:
ok. Let us take a closer look to determine whether this copyright is necessary or not

Reply:
We see no issues with that. Let's keep 2 copyrights in this and similar files